pennylane/devices/default_gaussian.py

```diff
@@ -779,13 +779,13 @@ def expand(self, S, wires):
     def expval(self, observable, wires, par):
 
         if observable == "PolyXP":
-            mu, cov = self._state
-            ev, var = self._observable_map[observable](
-                mu, cov, wires, self.wires, par, hbar=self.hbar
+            cov, mu = self._state
+            var, ev = self._observable_map[observable](
```

Comment:
Maybe the order of the expectation value and variance could be kept here and then we could leave them unchanged everywhere?

Reply:
I reset the order in the expectations back to ex,  var and therefore reversed it to the old order in `expval` and `var`.